pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -141,6 +142,9 @@ private void initPendingAckStore() {
                     }).exceptionally(e -> {
                         acceptQueue.clear();
                         changeToErrorState();
+                        pendingAckHandleCompletableFuture.completeExceptionally(
+                                new TransactionPendingAckException.TransactionPendingAckStoreProviderException(
+                                        e.getMessage()));
```

Comment:
e.getMessage() -> e.getCause() is much better.

Reply:
Thanks for your review. Good idea. I have addressed your comment. PTAL.